django_ipam/compat.py

```diff
@@ -10,12 +10,12 @@ def _is_subnet_of(a, b):
         try:
             # Always false if one is v4 and the other is v6
             if a._version != b._version:
-                raise TypeError("%s and %s are not of the same version".format((a, b)))
+                raise TypeError("%s and %s are not of the same version" % (a, b))

```

Comment:
`.format` -> `%`
Because
```
>>> "%s and %s".format("a", "b")
'%s and %s'
>>> "%s and %s" % ("a", "b")
'a and b'
>>> "{a} and {b}".format(a="a", b="b")
'a and b'
```